root/static/scripts/tests/release-editor/fields.js

```diff
@@ -183,3 +183,30 @@ fieldTest("tracks are set correctly when the cdtoc is changed", function (t, rel
     );
     t.ok(_.last(medium.tracks()).isDataTrack());
 });
+
+fieldTest("Tracks' time are changed correctly when inputting values in the medium tracklist editing form tab", function (t, release){
+    t.plan(8);
```

Comment:
Update the plan with the number of tests, or set it after `tests` using `plan(tests.length);`